packages/cli-lib/api/fileMapper.js

```diff
@@ -255,9 +255,23 @@ async function getDirectoryContentsByPath(portalId, path) {
   });
 }
 
+/**
+ * Deploy project
+ *
+ * @async
+ * @param {string} projectPath
+ * @returns {Promise}
+ */
+async function deployProject(portalId, projectPath) {
```

Comment:
Nit:  This could be someting like

```
const deployProject = async (portalId, projectPath) => http.post(portalId, {
  uri: `${FILE_MAPPER_API_PATH}/deploy/${encodeURIComponent(projectPath)}`,
});
```

Though not a big deal if you prefer your style

Reply:
I actually prefer that syntax as well, I was just matching the current pattern used throughout the file.